website/docs/general/events.md

```diff
@@ -18,16 +18,105 @@ that have a specific focus on Apache APISIX.
+* **Speaker** [Nicolas Fränkel](https://github.com/nfrankel)
+* **Dates** from 26-Apr-2022 to 28-Apr-2022
+* **Location** Lithuania
+
+#### [SciFiDevCon](https://www.scifidevcon.com/)
+
+* **Topic** Evolving your APIs
+* **Speaker** [Nicolas Fränkel](https://github.com/nfrankel)
+* **Dates** 1-May-2022 to 31-May-2022
+* **Location** Online
+
+#### [GeeCon](https://2022.geecon.org/)
+
+* **Topic** Evolving your APIs
+* **Speaker** [Nicolas Fränkel](https://github.com/nfrankel)
+* **Dates** from 11-May-2022 to 13-May-2022
+* **Location** Poland
+
+#### [Cloud Native Rejekts](https://cloud-native.rejekts.io/)
+
+* **Topic** Plugin Orchestration for an API Gateway
+* **Speaker** [Bobur Umurzokov](https://github.com/Boburmirzo)
+* **Dates** from 14-May-2022 to 15-May-2022
+* **Location** Spain
+
+#### [JavaCro Spring](https://2022spring.javacro.hr/eng/)
+
+* **Topic** Evolving your APIs
+* **Speaker** [Nicolas Fränkel](https://github.com/nfrankel)
+* **Dates** from 16-May-2022 to 17-May-2022
+* **Location** Croatia
+
+#### [JavaCro Spring](https://2022spring.javacro.hr/eng/)
+
+* **Topic** Chopping the monolith
+* **Speaker** [Nicolas Fränkel](https://github.com/nfrankel)
+* **Dates** from 16-May-2022 to 17-May-2022
+* **Location** Croatia
+
+#### [jPrime](https://jprime.io/)
+
+* **Topic** Evolving your APIs
+* **Speaker** [Nicolas Fränkel](https://github.com/nfrankel)
+* **Dates** from 25-May-2022 to 26-May-2022
+* **Location** Bulgaria
+
+#### [Java Day Istanbul](http://www.javaday.istanbul/)
+
+* **Topic** Evolving your APIs
+* **Speaker** [Nicolas Fränkel](https://github.com/nfrankel)
+* **Dates** from 28-May-2022 to 28-May-2022
+* **Location** Turkey
+
+#### [JNation](https://2022.jnation.pt/)
+
+* **Topic** Evolving your APIs
+* **Speaker** [Nicolas Fränkel](https://github.com/nfrankel)
+* **Dates** from 07-June-2022 to 07-June-2022
+* **Location** Portugal
+
+#### [WeAreDevelopers World Congress](https://worldcongress.dev/)
+
+* **Topic** Evolving your APIs
+* **Speaker** [Nicolas Fränkel](https://github.com/nfrankel)
+* **Dates** from 14-June-2022 to 14-June-2022
+* **Location** Germany
+
 #### [CloudLand](https://www.cloudland.org/de/home/)
 
-* **Topic:** Evolving your APIs
-* **Speaker:** [Nicolas Fränkel](https://github.com/nfrankel)
-* **Dates:** from 2022-06-29 to 2022-07-02
-* **Location:** Berlin, Germany
+* **Topic** Evolving your APIs
+* **Speaker** [Nicolas Fränkel](https://github.com/nfrankel)
+* **Dates** from 29-June-2022 to 02-July-2022
+* **Location** Germany
+
+#### [CloudLand](https://www.cloudland.org/de/home/)
+
+* **Topic** Chopping the monolith
+* **Speaker** [Nicolas Fränkel](https://github.com/nfrankel)
+* **Dates** from 29-June-2022 to 02-July-2022
+* **Location** Germany
+
+#### [Web Developer Conference](https://www.web-developer-conference.de/)
+
+* **Topic** Evolving your APIs
+* **Speaker** [Nicolas Fränkel](https://github.com/nfrankel)
+* **Dates** from 26-September-2022 to 28-September-2022
```

Comment:
Please keep consistent abbreviations.

Use `Sep` instead of `September `

Reply:
Good point! I fixed it too.